### ISSUE TITLE
core_1.Lifecycle.getInstance(...).emitTelemetry is not a function

### ISSUE THREAD

**Issue by KevinLaurence** [forcedotcom/sfdx-scanner#865]
**Describe the bug**
Upgraded to version 3.6.2 and now all runs result in the message:

`ERROR running scanner:run:  core_1.Lifecycle.getInstance(...).emitTelemetry is not a function`.

**To Reproduce**
Install or upgrade to version 3.6.2 and attempt to run the plugin.

**Expected behavior**
I would expect the plugin to execute without generating an error.

**Desktop:**
 - OS: Windows 10
 - Scanner Version 3.6.2
 - VS Code Version: 1.73.0 (user setup)
 - Commit: 8fa188b2b301d36553cbc9ce1b0a146ccb93351f
 - Date: 2022-11-01T15:34:06.111Z
 - Electron: 19.0.17
 - Chromium: 102.0.5005.167
 - Node.js: 16.14.2
 - V8: 10.2.154.15-electron.0
 - OS: Windows_NT x64 10.0.19044
 - Sandboxed: No

Has anyone seen this before?

**Comment by jfeingold35** [comment]
Hi @KevinLaurence , in order for us to help, we'll need a bit more information.
Are you running this through VSCode, or through `sfdx`? If the former, does it work when you call command directly from the CLI? If the latter, can you please tell us the output of `sfdx -v`?
Thanks!

**Comment by jfeingold35** [comment]
@KevinLaurence , could you also please provide the following?
1. How did you install `sfdx-scanner`? Did you use `sfdx plugins:install`/`sfdx plugins:update`, some installation using `npm`, or something else entirely?
2. What version did you update from?
3. What's the specific command you're running that triggers this issue?

**Comment by KevinLaurence** [comment]
Hi @jfeingold35,

Thanks for your quick response.

I am executing the `scanner:run` command at the command line in the Terminal window within VS Code.
```
sfdx scanner:run -t force-app\main\default\classes -f table
WARNING: We're continually improving Salesforce Code Analyzer. Tell us what you think! Give feedback at https://research.net/r/SalesforceCA
ERROR running scanner:run:  core_1.Lifecycle.getInstance(...).emitTelemetry is not a function
```
I am on the following version of sfdx: `sfdx-cli/7.174.0 win32-x64 node-v16.17.1`

The plugin was updated within the last week by running `sfdx update` and the previous version was 2.13.9.

**Comment by jfeingold35** [comment]
@KevinLaurence , that's bizarre. Could you please try uninstalling and reinstalling the plugin, and let us know if that fixes it?

**Comment by KevinLaurence** [comment]
@jfeingold35
Uninstalled the plugin:
```
sfdx plugins:uninstall @salesforce/sfdx-scanner
Uninstalling @salesforce/sfdx-scanner... done
```
Shut down VS Code. Restarted VS Code.

Reinstalled the plugin. Executed the run command. Same error:
```
sfdx plugins:install @salesforce/sfdx-scanner
Polling for new version(s) to become available on npm... done
Successfully validated digital signature for @salesforce/sfdx-scanner.
Finished digital signature check.
warning "@salesforce/sfdx-scanner > ts-node@10.9.1" has unmet peer dependency "@types/node@*".
Installing plugin @salesforce/sfdx-scanner... installed v3.6.2
PS > sfdx scanner:run -t force-app\main\default\classes -f table
WARNING: We're continually improving Salesforce Code Analyzer. Tell us what you think! Give feedback at https://research.net/r/SalesforceCA
ERROR running scanner:run:  core_1.Lifecycle.getInstance(...).emitTelemetry is not a function
```
I'm not certain if the first warning, `ts-node@10.9.1" has unmet peer dependency "@types/node@*"`, is significant or not.

**Comment by jfeingold35** [comment]
Could you please try uninstalling and reinstalling the plugin again, and then running the same steps that you would normally run outside of VSCode, in a terminal window?
And could you please let us know what shell you're using?

**Comment by jfeingold35** [comment]
@KevinLaurence , Just wanted to check in with you about this one. Have you had a chance to do the test we advised? Thanks!

**Comment by KevinLaurence** [comment]
@jfeingold35 Yes, I have uninstalled the plugin and the SFDX CLI, and then attempted to install them again. The CLI was installed correctly, and upgraded to the latest version (7.176.1). The plug-in fails to install correctly now (yarn add generates an error), but I suspect it's because of constraints imposed by my IT department. I will follow up with them next. Thank you for your help.

**Comment by jfeingold35** [comment]
Hi @KevinLaurence , we've merged PR #896 , which we anticipate should fix this issue. We'll be releasing that change next week.

**Comment by jfeingold35** [comment]
@KevinLaurence , as mentioned, we believe a fix that went into the 3.7.1 release should have fixed this issue. Could you please update to that version and let us know?
Thanks!

**Comment by KevinLaurence** [comment]
Hi @jfeingold35 I have installed the latest version and the plugin is working as expected now. Thank you for your help.

**Comment by jfeingold35** [comment]
Resolved by PR #896 , in release 3.7.1.
